lib/api/v3/queries/schemas/query_schema_representer.rb

```diff
@@ -105,6 +105,13 @@ def self.filters_schema
                  has_default: true,
                  visibility: false
 
+          schema :timeline_zoom_level,
+                 type: 'String',
+                 required: false,
+                 writable: true,
+                 has_default: true,
+                 visibility: false
```

Comment:
I wouldn't do it for the sake of the front end but for the sake of the API itself. From my understanding, following HAL we should strive to link as much as possible and by that allow the client to refer to the documentation as little as possible.